### ISSUE TITLE
Second-wind phenomenon

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#11647]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
PMID: 39318660 The "second-wind" phenomenon is experienced by individuals with McArdle's disease. This phenomenon, occurring approximately 10 minutes after the onset of exertion, denotes a sudden and significant improvement in aerobic exercise tolerance after a brief rest period, following the development of muscle stiffness and myalgias, enabling the person to continue physical activity with a resolution of symptoms triggered at the start of exertion. The "second wind" can be explained by increased blood flow, optimized delivery of free fatty acids with concurrent activation of their metabolism, and increased utilization of glucose. However, pre-exercise glucose intake can prevent the onset of this phenomenon.

PMID: 20301518 Second-wind phenomenon, i.e., improvement in exercise-induced muscle cramps and/or pain after a brief rest period when exercise intensity is reduced, or after the first (~) ten minutes of continuous exercise at the same constant, moderate intensity.

**Parent term (use hpo.jax.org/app)**
Exercise-induced muscle cramps 
HP:0003710

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Glycogen storage disease due to muscle glycogen phosphorylase deficiency  ORPHA:368

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6001543
- label: Second-wind phenomenon
- parents: [Abnormal muscle physiology (HP:0011804)](https://hpo.jax.org/app/browse/term/HP:0011804)
- pmid: PMID:39318660|PMID:20301518
- def: The second-wind phenomenon occurs approximately 10 minutes after the onset of exertion, denotes a sudden and significant improvement in aerobic exercise tolerance after a brief rest period, following the development of muscle stiffness and myalgias, enabling the person to continue physical activity with a resolution of symptoms triggered at the start of exertion.
- comment: The second wind phenomenon is experienced by individuals with McArdle's disease and can be explained by increased blood flow, optimized delivery of free fatty acids with concurrent activation of their metabolism, and increased utilization of glucose. However, pre-exercise glucose intake can prevent the onset of this phenomenon.
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11647
- synonyms:
